lib/group_consumer_stream.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var ReadableStream = require('readable-stream').Readable;
+var Promise        = require('./bluebird-configured');
+var _              = require('lodash');
+var GroupConsumer  = require('./group_consumer');
+
+function GroupConsumerStream(options) {
+  this.readableStream = new ReadableStream({objectMode: true});
+  this.readableStream._read = function() {};
+
+  this.groupConsumer = new GroupConsumer(options);
+  this._end = _.bind(this.groupConsumer.end, this.groupConsumer);
+  this.init = _.bind(this.groupConsumer.init, this.groupConsumer);
+  this.fetchOffset = _.bind(this.groupConsumer.fetchOffset, this.groupConsumer);
+  this._commitOffset = _.bind(this.groupConsumer.commitOffset, this.groupConsumer);
+}
+
+module.exports = GroupConsumerStream;
+
+GroupConsumerStream.prototype._read = function () {};
+
+// letting users set handler doesn't make sense in the context of a stream
+GroupConsumerStream.prototype.getStream = function (strategies) {
+    var self = this;
+
+    var handler = function (messageSet, topic, partition) {
+      return Promise.all(messageSet.map(function(m) {
```

Comment:
I kind of had to commit the offsets since I am providing a simplified interface via a readable stream.